### DISCUSSION TITLE
Problem with Companion PI

### DISCUSSION THREAD

**Discussion by mariospinaci** [cpvalente/ontime#958] [Discussion]
Good evening, first of all congratulations of the application.
I am writing because a curious problem occurred to me.
On a Raspberry 4 8GB I installed the PI Companion, then later I installed OnTime using the docker. The iteration between the two software works, the only problem is that all Ontime variables are empty.
What could it depend on?

Thank you

**Comment by cpvalente** [comment]
> Good evening, first of all congratulations of the application.
> 
> I am writing because a curious problem occurred to me.
> 
> On a Raspberry 4 8GB I installed the PI Companion, then later I installed OnTime using the docker. The iteration between the two software works, the only problem is that all Ontime variables are empty.
> 
> What could it depend on?
> 
> 
> 
> Thank you

Hi, can you please clarify which version of companion do you have?
Ontime v3 is only compatible with the companion module v4

**Comment by lukestein** [comment]
A datapoint for OP: I'm running Docker Ontime 3.0.0 and had the same issue you note (companion connection but not actually working) with the latest stable release of CompanionPi.

Needed to update to latest beta CompanionPi to get the updated module that supports ontime 3.0.0

Try `sudo companion-update` and update to latest beta? **Backup companion first. Backup!!!**

**Reply by cpvalente** [reply]
Thank you for this
I will try and follow up with the bitfocus team and see if this is a module problem
